package.json

```diff
@@ -39,6 +39,7 @@
     "sync-request": "3.0.1",
     "typescript": "1.8.7",
     "validator": "3.33.0",
-    "node-uuid": "1.4.6"
+    "node-uuid": "1.4.6",
+    "nock": "9.0.2"
```

Comment:
These aren't task specific dependencies.  Of course, "real" task specific dependencies are scoped to the task.

But we should be clear that this repo has an explicit goal of sharing and driving consistent engineering.  In that spirit we should drive to share engineering which is by definition a dev dependency.